### PR TITLE
Adding parallel_dd test [V5]

### PR DESCRIPTION
**Pull request by poojabs7**
This test measures the performance of writing and reading multiple streams of files onto the files system.

Signed-off-by: poojabs <pooja@oc2013156016.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@poojabs7 job log please

**Comment by poojabs7** [comment]
[parallel_dd_joblog.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/633137/parallel_dd_joblog.txt)
Attached job log.

File: io/disk/parallel_dd.py.data/parallel_dd.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+setup:
+    filesystem: !mux
+        default:
+            disk:
+    mb: !mux
+        default:
+            megabytes: 100
+    blocks: !mux
+        first:
+            blocks:
```
Comment (PraveenPenguin) [inline io/disk/parallel_dd.py.data/parallel_dd.yaml]: can you please pass some default value of all these option , i face hard time to pass the value as it is string or integer  or you an add this README  but i prefer yaml

**Comment by poojabs7** [comment]
Raised new PR.
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/214

**Pull request closed without merging**